scripts/requirements.txt

```diff
@@ -17,6 +17,9 @@ virtualenv
 # Infineon
 -r requirements.infineon.txt
 
+# Infineon
```

Comment:
TI?

Reply:
Sorry, copy/paste error in the heat of the moment, should be fixed now 😄